docs/rest.md

```diff
@@ -90,7 +93,11 @@ wrong, in those cases, the following return value is used:
 
 ```javascript
 {
- "result": {},            // may contain more details, for debugging
+ "result": {
+   "message": "human-friendly description of the cause of the error",
+   "kind": "store-error",  // one of a list of kinds (TBD), only present iff "value" is present
+   "value": {"...": "..."} // kind-specific object, as required
```

Comment:
An example how such an error kind/value might look like would be nice. Maybe something like "kind": "oserror", value: {"errno": "EACCESS"} (if that makes sense and fits the planed use)